### PR TITLE
Send server render times to tracking server

### PR DESCRIPTION
**Pull request by ajacksified**
This will post the initial server render timings to statsd (via hivemind.)

:eyeglasses: @danehansen or @curioussavage

### CODE REVIEW

**Comment by danehansen** [comment]
:fish:

File: assets/js/client.es6.js [hunk 1]
```diff
@@ -69,7 +71,10 @@ function refreshToken (app, tries=0) {
   });
 }
 
+
+
```
Comment (curioussavage) [inline assets/js/client.es6.js]: nit: extra whitespace here

File: src/routes.jsx [hunk 2]
```diff
@@ -434,12 +438,12 @@ function routes(app) {
     this.props = props;
   }
 
-  app.router.get('/comments/:listingId', commentsPage);
-  app.router.get('/comments/:listingId/:listingTitle', commentsPage);
-  app.router.get('/r/:subreddit/comments/:listingId', commentsPage);
-  app.router.get('/r/:subreddit/comments/:listingId/:listingTitle', commentsPage);
+  app.router.get('comments.listingid', '/comments/:listingId', commentsPage);
```
Comment (curioussavage) [inline src/routes.jsx]: this is for organizing routes in stats/graphite?

**Comment by curioussavage** [comment]
mostly just curious, but looks good to me too!

File: src/routes.jsx (hunk 2 above)
Comment (ajacksified) [inline src/routes.jsx]: Yup. As it happens, we can also generate urls like this: https://github.com/alexmingoia/koa-router#named-routes

**Pull request merged**